cmd/nebula-cert/main.go

```diff
@@ -71,6 +71,8 @@ func main() {
 		err = printCert(args[1:], os.Stdout, os.Stderr)
 	case "verify":
 		err = verify(args[1:], os.Stdout, os.Stderr)
+	case "qr":
+		err = qr(args[1:], os.Stdout, os.Stdin)
```

Comment:
Just because you can, doesn't mean you should....

Arbitrary QR encoding is pretty far outside the scope of nebula-cert and definitely feels like a bolted on feature.

Perhaps adding the `-qr` flag on `print` mode fulfils the missing use case?

Reply:
Yes. That is way better. Thank you!